libraries/AP_Notify/Buzzer.h

```diff
@@ -66,11 +72,12 @@ class Buzzer: public NotifyDevice
 
     /// buzzer_flag_type - bitmask of current state and ap_notify states we track
     struct buzzer_flag_type {
+        uint8_t vehicle_lost        : 1;     //1 Vehicle Lost
```

Comment:
I don't think this is actually needed, but it does follow the way it was implemented (there's already other vars that weren't needed)